pkgs/development/tools/literate-programming/Literate/dub-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dependencies": {}
+}
```

Comment:
This could be inlined or even just be the default of buildDubPackage.

Reply:
It *could* be inlined, but I don't think many packages have 0 dependencies. If we have a default, it won't show a missing attribute error, which is worse IMO.